src/Model/EndpointRegistry.php

```diff
@@ -8,6 +8,11 @@
 use Muffin\Webservice\AbstractDriver;
 use RuntimeException;
 
+/**
+ * EndpointRegistry
+ *
+ * @deprecated 2.0.0 The registry has been deprecated, please use EndpointLocator instead.
+ */
 class EndpointRegistry
```

Comment:
Doesn't the usage of `EndpointRegistry` need to be updated anywhere in the plugin itself? I do see any such changes in the diff.

Reply:
Strangely not. It seems that it's only used in the `boostrap.php` to register the repository type with the FactoryLocator. Which is why I think the `bootstrap.php` file is deprecated really. Removing it and the `EndpointRegistry` would break BC though.